src/styles/blocks/container.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  margin: 0 auto;
+  max-width: 1020px;
+  padding-inline: 20px;
+}
+
+@media (min-width: 596px) {
+  .container {
+    padding-inline: 72px;
```

Comment:
It is recommended to use `padding` instead of the `padding-inline` property, as `padding-inline` has limited browser support.